lib/init/grass.py

```diff
@@ -2455,6 +2221,15 @@ def validate_cmdline(params):
     # without --exec (usefulness to be evaluated).
 
 
+def find_python_packages():
+    if os.path.exists(gpath("etc", "python")):
+        pythonpath = gpath("etc", "python")
+        sys.path.append(pythonpath)
+        # now we can import stuff from grass package
+    else:
+        raise RuntimeError("Python package grass is missing")
```

Comment:
> Does this mean that each time this function is called, we add pythonpath to sys.path ?

It is a function in a sort of standalone script meant for one use, so I'm not much concerned about it here. I did not consider how costly (or inexpensive) is to just check... 

> Also, pythonpath (or PYTHONPATH) has a meaning in Python, and it is used as a variable name here, it might be confusing when searching the code later.

Agreed. I don't like the name either. I got it from the old code.

Reply:
Name changed in 0082201abf982545855c8f96c962a51062c0d579.